### ISSUE TITLE
Add possibility to disable certificate verification

### ISSUE THREAD

**Issue by glc-mvion** [aholachek/bundle-wizard#1]
Hello ! :wave: 

I found your tool very useful but when I wanted to try it on a development website on my machine I was greeted the error I pasted at the end of the issue.

Thank you in advance (:

<details>
<summary>Terminal log</summary>

```
❯ npx bundle-wizard

🧙‍  Welcome to bundle-wizard

✔ Which site would you like to analyze? · https://localhost:4200


🤖  Recording page load info for https://localhost:4200 ...


❌  Unable to fetch website data

Error: net::ERR_CERT_INVALID at https://localhost:4200
    at navigate (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/FrameManager.js:120:37)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async FrameManager.navigateFrame (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/FrameManager.js:94:17)
    at async Frame.goto (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/FrameManager.js:406:12)
    at async Page.goto (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/Page.js:672:12)
    at async downloadCoverage (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/src/download-coverage.js:106:3)
    at async main (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/src/index.js:27:28)
  -- ASYNC --
    at Frame.<anonymous> (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/helper.js:111:15)
    at Page.goto (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/Page.js:672:49)
    at Page.<anonymous> (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/node_modules/puppeteer-core/lib/helper.js:112:23)
    at downloadCoverage (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/src/download-coverage.js:106:14)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async main (/home/matthieu/.npm/_npx/16021/lib/node_modules/bundle-wizard/src/index.js:27:28)

```
</details>

**Comment by aholachek** [comment]
Thank you for the detailed report! 

I just released `0.1.14`, that adds a new [`ignoreHTTPSErrors` flag](https://github.com/aholachek/bundle-wizard#ignorehttpserrors-flag). Please try it and let me know if it fixes your problem.

**Comment by glc-mvion** [comment]
It works perfectly ! Thanks you very much for the quick fix :bowing_man:
